frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte

```diff
@@ -3,6 +3,7 @@
   import {Button} from '$lib/components/button';
   export let password = '';
   export let autocomplete = '';
+  export let id = password;
```

Comment:
The default value should be something else than the value of the password prop. Did you mean to set it to `"password"`?